yb-voyager/cmd/callhome_payload_builder.go

```diff
@@ -289,3 +293,70 @@ func getAnonymizedDDLs(sourceDBConf *srcdb.Source) []string {
 
 	return anonymizedDDLs
 }
+
+
+// ============================export schema callhome payload information============================
+
+func buildCallhomeSchemaOptimizationChanges() []callhome.SchemaOptimizationChange {
+	if schemaOptimizationReport == nil {
+		return nil
+	}
```

Comment:
Q: is this just to safeguard or there is possibility to hit `nil`

consider adding log before return